svo (aliases, triggers).xml

```diff
@@ -21192,21 +21192,9 @@ svo.parseplistend()</script>
 				</Trigger>
 				<Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
 					<name>svo started dragonform (lesser)</name>
-					<script>svo.defs.started_dragonform()
-
-local t = {
-   ["red"] = 'dragonfire',
-   ["black"] = 'acid',
-   ["silver"] = 'lightning',
-   ["gold"] = 'psi',
-   ["blue"] = 'ice',
-   ["green"] = 'venom'
-}
-
-if not t[multimatches[2][2]] then return end
+					<script>svo.valid.dragonformingcolour(multimatches[2][2])
```

Comment:
There is no reason to add this in there. The colour for breath weapon gets set on the middle dragonforming trigger hence the reason for the start dragonforming so it will recognize the second trigger and then complete with the third one.